apps/deep-clone-main/src/utils/useInstallationParameters.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import { BaseAppSDK } from '@contentful/app-sdk';
+import { AppInstallationProps, KeyValueMap } from 'contentful-management';
+
+export const useInstallationParameters = (sdk: BaseAppSDK) => {
+  const [parameters, setParameters] = useState<KeyValueMap>(sdk.parameters.installation);
+
+  useEffect(() => {
+    const fetch = async () => {
+      const newParameters = await fetchParameters(sdk);
+      setParameters(newParameters);
+    };
+    fetch();
+  }, [sdk]);
+
+  return parameters;
+};
+
+const fetchParameters = async (sdk: BaseAppSDK): Promise<KeyValueMap> => {
+  try {
+    if (!sdk.ids.organization || !sdk.ids.app || !sdk.ids.space) {
+      throw new Error('Required SDK IDs not available');
+    }
+
+    const appInstallation = await sdk.cma.appInstallation.getForOrganization({
+      appDefinitionId: sdk.ids.app,
+      organizationId: sdk.ids.organization,
+    });
+
+    const currentInstallation = appInstallation.items.find((installation: AppInstallationProps) => installation.sys.space.sys.id === sdk.ids.space);
+
+    if (currentInstallation?.parameters) {
+      return currentInstallation.parameters as KeyValueMap;
+    }
+  } catch (error) {
+    console.warn('Failed to fetch fresh parameters from CMA:', error);
```

Comment:
💭 How we want to handle errors? I don't think we should console log something here. Perhaps a notifier saying "Something went wrong. Please refresh" ?